pkg/filesystem/stargz_adaptor.go

```diff
@@ -36,6 +36,10 @@ func (fs *Filesystem) StargzEnabled() bool {
 	return fs.stargzResolver != nil
 }
 
+func (fs *Filesystem) ReferrerDetectEnabled() bool {
```

Comment:
This function should not locate at stargz adopter.

Reply:
Fixed.